src/java.desktop/macosx/native/libawt_lwawt/awt/CGraphicsDevice.m

```diff
@@ -35,6 +35,7 @@
 #define DEFAULT_DEVICE_HEIGHT 768
 #define DEFAULT_DEVICE_DPI 72
 
+static NSInteger architecture = -1;
```

Comment:
This is optional, but since the `architecture` variable is only used in `isValidDisplayMode`, it can be declared [static](https://stackoverflow.com/a/4965145) inside the method itself.